database/migrations/2015_11_23_092743_add_columns_to_users_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddColumnsToUsersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function(Blueprint $table){
```

Comment:
Just checked the branch I just pushed, in 2015_11_23_092743_add_columns_to_users_table.php. There is space before the open parenthesis for the two cases.